documentation/specs/proposed/BuildCheck-Architecture.md

```diff
@@ -35,7 +35,7 @@ One example of rich data that might be helpful for internal analyses is [`Projec
 
 ## Execution Modes
 
-**Replay Mode** - so that users can choose to perform analyses post build, without impacting the performance of the build. And so that some level of analysis can be run on artifacts from builds produced by older versions of MSBuild.
```

Comment:
Let's not remove completely - it is still valid and usefull scenario to run rellay analysis in a newer version that might have additional builtin checks. Though the formulation should mention that the binlog will need to be produced by msbuild from NET 9.0 / VS 17.12 or newer - older won't be supported